freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/user/UserV1Endpoint.java

```diff
@@ -52,4 +52,11 @@
     @ApiOperation(value = UserOperationDescriptions.SYNC_OPERATION_STATUS, notes = UserNotes.USER_NOTES, produces = MediaType.APPLICATION_JSON,
             nickname = "getSyncOperationStatusV1")
     SyncOperationStatus getSyncOperationStatus(@NotNull @QueryParam("operationId") String operationId);
+
+    @GET
+    @Path("getKeytab")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = UserOperationDescriptions.GET_KEYTAB, notes = UserNotes.USER_NOTES, produces = MediaType.APPLICATION_JSON,
+            nickname = "getKeytabV1")
+    String getKeytab(@NotNull @QueryParam("environment") String environment, @QueryParam("user") String user);
```

Comment:
if the query params are CRNs, then let's include CRN in the names. e.g., environmentCrn